src/common/components/ImageVector/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { getAsset } from "@/core/assets"
+import { useEffect, useRef } from "react";
+
+let _wrapper: HTMLDivElement;
+
+export default function (props: any): JSX.Element {
+  const wrapper = useRef(null);
+  const image = getAsset(props.kind, props.identifier)?.data;
```

Comment:
`props.kind` not required as it's implicitly an image ... directly replace with `getAsset('image', ...`